spec/requests/atmosphere/api/clew_spec.rb

```diff
@@ -48,6 +48,7 @@
                       [{"id"=>appl.id,
                         "name"=>appl.name,
                         "appliance_set_id"=>appl.appliance_set_id,
+                        "appliance_type_id" => appl.appliance_type_id,
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.